ci/jenkins/test-vmc.sh

```diff
@@ -260,7 +260,7 @@ function setup_cluster() {
         do
             sleep 1m
             echo "=== Get node (try for 1m) ==="
-            mdNum="$(kubectl get node | grep -c ${CLUSTER}-md)"
+            mdNum="$(kubectl get node | grep -c ${CLUSTER}-m)"
```

Comment:
May I know why this change is necessary? Is it because capv changed the naming pattern for the machine deployment VMs.

Reply:
CAPV will rename the cluster to "-m" for longer node name. I have update this code to only count worker node number.